### PR TITLE
SIWPA-34

### PR DESCRIPTION
**Pull request by Blaine-Mason**
- Add Charts on admin dashboard 
- Add the respective dependencies to package.json

### CODE REVIEW

File: src/components/AdminDashboard.vue [hunk 1]
```diff
@@ -28,16 +46,129 @@
...
 <script>
-
- import PendingListingModule from "./PendingListingModule.vue";
+ import PendingListingModule from "./PendingListingModule.vue"
  import ContactInboxModule from "./ContactInbox.vue"
-export default {
-  name: "AdminDashboard",
-   
-  components: {
+ import Vue3ChartJs from '@j-t-mcc/vue3-chartjs'
```
Comment (jventura1738) [inline src/components/AdminDashboard.vue:51]: Not really major, but if you choose to use `;` be consistent and use for all lines.  Just a styling suggestion no changes needed

Comment (jventura1738) [reply src/components/AdminDashboard.vue:51]: ^^ Same for `'` and `"`

**Review by jventura1738 — APPROVED** [review]
✅ @jacobduncan00 confirm pls

File: src/components/AdminDashboard.vue (hunk 1 above)
Comment (jacobduncan00) [reply src/components/AdminDashboard.vue:51]: I would agree with this, like Justin said just a minor change, I have my prettier config do this so I don't have to worry about it.

**Review by jacobduncan00 — APPROVED** [review]

File: src/components/AdminDashboard.vue (hunk 1 above)
Comment (Blaine-Mason) [reply src/components/AdminDashboard.vue:51]: Noted, will do from now on. Thank you.

**Pull request merged**